test/XRP/helpers/xrp-test-utils.ts

```diff
@@ -29,6 +37,70 @@ export default class XRPTestUtils {
     }
     return paymentXrpTransactions
   }
+
+  /**
+   * Generates a random wallet and funds using the XRPL Testnet faucet.
+   */
+  static async randomWalletFromFaucet(): Promise<Wallet | undefined> {
```

Comment:
since we can't do anything with `undefined` can this function just throw if something goes awry?

Reply:
throws

Reply:
since it throws and we can't do anything with `undefined`, can we remove it from the return type?